Squablr/Models/User.h

```diff
@@ -13,6 +13,7 @@ NS_ASSUME_NONNULL_BEGIN
 @interface User : PFObject<PFSubclassing>
 @property (nonatomic, strong) NSString *userID;
 @property (nonatomic, strong) PFUser *name;
+//@property (nonatomic, strong) NSString *email;
```

Comment:
Why are we commenting this out?

Reply:
Not sure, think I was getting ahead of myself